pymongo/monitoring.py

```diff
@@ -615,6 +625,14 @@ def database_name(self) -> str:
         """
         return self.__db
 
+    @property
+    def server_connection_id(self) -> Optional[ObjectId]:
```

Comment:
ObjectId? I thought connectionId was an integer?

Reply:
Good catch. The typecheck must have missed it because the tests use `assertIsInstance` instead of a `mypy`-friendly typecheck.